app/src/main/java/com/socketmobile/stockCount/ui/OptionsActivity.kt

```diff
@@ -31,6 +53,48 @@ class OptionsActivity : AppCompatActivity() {
             val i = Intent(Intent.ACTION_VIEW, Uri.parse(getString(R.string.sdk_github_url)))
             startActivity(i)
         }
+        buyScannerButton.setOnClickListener {
+            val progressDialog = ProgressDialog(this)
+            progressDialog.show()
+            val country = Locale.getDefault().country
+
+            if (country == "CN") {
```

Comment:
Why is there a specific case for China? Can we explain this in a comment? Also it would be better to have `CN` string being replaced by a constant.

Reply:
Now there's no chinese link that is registered to refresion, so using link directly.
So I had to hard coding for china.